src/Utilities/VariableTemplates/flattened_tup_chain.jl

```diff
@@ -33,3 +34,54 @@ function flattened_tup_chain(
     Iterators.flatten |>
     collect
 end
+flattened_tup_chain(::AbstractVars{S}) where {S} = flattened_tup_chain(S)
+
+export flattened_named_tuple
```

Comment:
Please move all `export` statements to the top of the module defining file (`VariableTemplates.jl` in this case).